ui/app/AppLayouts/Profile/panels/ProfileShowcasePanel.qml

```diff
@@ -0,0 +1,352 @@
+        readonly property int contentSpacing: 12
+        readonly property int strokeMargin: 2
+    }
+
+    contentItem: ColumnLayout {
+        id: layout
+        spacing: d.contentSpacing
+
+        StatusBaseText {
+            Layout.fillWidth: true
+            text: qsTr("In showcase")
+            color: Theme.palette.baseColor1
+            font.pixelSize: 13
+            font.weight: Font.Medium
+        }
+
+        StatusListView {
+            id: showcaseItemsListView
+            Layout.fillWidth: true
+            Layout.preferredHeight: count ? contentHeight : targetDropArea.height
+            model: showcaseModel
+            interactive: false
+
+            displaced: Transition {
+                NumberAnimation { properties: "x,y"; easing.type: Easing.OutQuad }
+            }
+
+            delegate: DropArea {
+                id: showcaseDelegateRoot
+
+                property int visualIndex: index
+
+                ListView.onRemove: SequentialAnimation {
+                    PropertyAction { target: showcaseDelegateRoot; property: "ListView.delayRemove"; value: true }
+                    NumberAnimation { target: showcaseDelegateRoot; property: "scale"; to: 0; easing.type: Easing.InOutQuad }
+                    PropertyAction { target: showcaseDelegateRoot; property: "ListView.delayRemove"; value: false }
+                }
+
+                width: ListView.view.width
+                height: showcaseDraggableDelegateLoader.item ? showcaseDraggableDelegateLoader.item.height : 0
+
+                keys: ["x-status-draggable-showcase-item"]
+
+                onEntered: function(drag) {
+                    const from = drag.source.visualIndex
+                    const to = showcaseDraggableDelegateLoader.item.visualIndex
+                    if (to === from)
+                        return
+                    showcaseModel.move(from, to, 1)
+                    drag.accept()
+                }
+
+                Loader {
+                    id: showcaseDraggableDelegateLoader
+                    width: parent.width
+                    sourceComponent: root.showcaseDraggableDelegateComponent
+
+                    property var modelData: model
+                    property var dragParentData: root
+                    property int visualIndexData: index
+                }
+            }
+
+            // overlaid at the bottom of the listview
+            DropArea {
+                id: targetDropArea
+                width: parent.width
+                height: d.defaultDelegateHeight
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: showcaseModel.count ? Style.current.halfPadding : 0
+                keys: ["x-status-draggable-showcase-item-hidden"]
+
+                ShapeRectangle {
+                    width: parent.width - d.strokeMargin
+                    height: parent.height - d.strokeMargin
+                    anchors.centerIn: parent
```

Comment:
I'd say that 
```qml
anchors.fill: parent
anchors.margins d.strokeMargin
```
is a bit more readable, but that's probably just a question of style 🙂

Reply:
Fixed